app/src/ui/add-repository/create-repository.tsx

```diff
@@ -434,6 +463,25 @@ export class CreateRepository extends React.Component<
     )
   }
 
+  private renderReadmeOverwriteWarning() {
+    if (
+      this.state.createWithReadme === false ||
+      this.state.readMeExists === false
+    ) {
+      return null
+    }
+
+    return (
+      <Row className="warning-helper-text">
+        <Octicon symbol={OcticonSymbol.alert} />
+        <p>
+          This directory contains a <Ref>ReadMe.md</Ref> file already. Checking
```

Comment:
Seeing as we're using `README` fully capitalized in the label could we make this be `README.md` as well?

```suggestion
          This directory contains a <Ref>README.md</Ref> file already. Checking
```

That would also be in line with what we're using on GitHub.com

<img width="341" alt="image" src="https://user-images.githubusercontent.com/634063/49441813-e35ee600-f7c7-11e8-9ed1-804eb6393669.png">